js/models/payment-request.js

```diff
@@ -0,0 +1,35 @@
+var app = app || {};
+
+app.models = app.models || {};
+
+app.models.PaymentRequest = (function() {
+
+	'use strict';
+
+	return Backbone.Model.extend({
+        idAttribute: 'id',
+
+        defaults: {
+            currency: '',
+            address: '',
+            confirmed: '',
+            amount: '',
+            timestamp: (new Date).getTime()
```

Comment:
This will set the default timestamp equal to the datetime when the app is loaded. The backbone model `defaults` attribute can be an object or a function:
http://backbonejs.org/#Model-defaults

Example:
```js
defaults: function() {
    return {
        currency: '',
        address: '',
        confirmed: '',
        amount: '',
        timestamp: (new Date).getTime()
    };
}
```
Which will be run any time an instance is created from the model prototype.